authors/apps/authentication/tests/test_social_auth.py

```diff
@@ -59,8 +61,8 @@ def test_facebook_oauth(self):
     def test_google_auth(self):
         """Test Google Oauth"""
         self.data = {
-            "access_token": os.environ['GOOGLE_ACCESS_TOKEN'],
-            "access_token_secret": os.environ['TWITTER_ACCESS_TOKEN_SECRET'],
```

Comment:
@BrianSerem, please remove the fixes that do not contribute to the PT board story. It is better to have this in a separate PR.

Reply:
Okay, I have removed the unrelated changes.